### PR TITLE
Introduce healthcheck link

### PR DESCRIPTION
**Pull request by johnschoeman**
Why:
We would like to optionally include an external link to a healthcheck or
symptom checker web application.

This commit:
Introduces the component and basic configuration for a healthcheck link.

|healthcheck link|
|---|
|Simulator Screen Shot - iPhone 12 Pro - 2020-12-18 at 08 37 21|

### CODE REVIEW

File: src/Home/ShareLink.tsx [hunk 1]
```diff
@@ -17,30 +24,31 @@ import {
...
     try {
       await Share.share({
         message: t("home.bluetooth.share_message", {
           applicationName,
-          appDownloadLink: configuration.appDownloadLink,
+          appDownloadUrl,
         }),
       })
     } catch (error) {
-      throw new Error(error)
```
Comment (johnschoeman) [inline src/Home/ShareLink.tsx:36]: Did not seem correct to me to crash the app if the link is not provided by accident.

File: src/Home/index.tsx [hunk 2]
```diff
@@ -69,7 +73,10 @@ const Home: FunctionComponent = () => {
         ) : (
           <EscrowVerificationFlowButton />
         )}
-        <ShareLink />
+        {healthAuthorityHealthCheckUrl && (
+          <HealthCheckLink healthCheckUrl={healthAuthorityHealthCheckUrl} />
+        )}
+        {appDownloadUrl && <ShareLink appDownloadUrl={appDownloadUrl} />}
```
Comment (johnschoeman) [inline src/Home/index.tsx:79]: pulled this up to keep a consistent layer of abstraction for the view logic on the dashboard.

File: src/factories/configurationContext.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { Factory } from "fishery"
 import { Configuration } from "../ConfigurationContext"
 
 export default Factory.define<Configuration>(() => ({
-  appDownloadLink: "appDownloadLink",
+  appDownloadUrl: "appDownloadUrl",
```
Comment (johnschoeman) [inline src/factories/configurationContext.ts:5]: changed Link to Url to be consistent with the rest of the provided url variable names

**Review by mattThousand — APPROVED** [review]
LGTM!

**Pull request merged**